lib/charms/operator_libs_linux/v1/snap.py

```diff
@@ -264,6 +287,63 @@ def unset(self, key) -> str:
         """
         return self._snap("unset", [key])
 
+    def start(self, services: Optional[List[str]] = None, enable: Optional[bool] = False) -> None:
+        """Starts a snap's services.
+
+        Args:
+            services (list): (optional) list of individual snap services to start (otherwise all)
+            enable (bool): (optional) flag to enable snap services on start. Default `false`
+        """
+        options = []
+        if enable:
+            options.append("--enable")
+
+        self._snap_daemons("start", services, options)
```

Comment:
are '--enable' and '--disable' the only possible options one can pass to snap cmd? If not, why is it the only one we wrap? 
How does the next PR extend this pattern?

Reply:
Mmm ok, yes. An idea to have less repetitions all over would be:
`cmd = ['cmd str', 'option str'] if option else ['cmd str']
`self._snap_daemons(cmd, services)`

instead of accepting `options=Iterable[str]` when in practice it's only ever going to be a single string (or None)